content/en/Features/service2service.md

```diff
@@ -1,32 +1,82 @@
+seamlessly assimilate into the installed service mesh. Combined with Linkerd's official principles of 
+[keep it simple, minimize resource requirements and just work](https://linkerd.io/design-principles/#), this means that
+most development should not even notice the presence of Linkerd and upkeep should be nigh non-existent.
 
-- Every service also gets an internal-only domain name that can be used by other
-  services to connect to it. The name is `<module-name>.<service-name>`. Requests
-  sent to this domain will automatically get load balanced b/w all healthy
-  containers.
+### Linkerd Viz
+For monitoring, Linkerd comes with an optional dashboard solution known as [Linkerd Viz](https://linkerd.io/2.11/features/dashboard/).
+It is not installed by default due to the non-negligible resource requirement (it installs Prometheus and Grafana
+as well as a few smaller tools), but can be easily added with the following commands: 
+
+```shell
+curl -fsL https://run.linkerd.io/install | sh   # Install linkerd CLI
+linkerd viz install | kubectl apply -f -        # Install linkerd monitoring stack
+linkerd viz dashboard                           # open the dashboard
+```
+
+### Service Discovery
+Every service also gets an internal-only domain name that can be used by other
+services to connect to it. The name is `<module-name>.<service-name>`. Requests
+sent to this domain will automatically get load balanced b/w all healthy
+containers.
 
 For example, the following service, will be available at `app.hello-world` from any other service in the same environment.
 
 {{< highlight yaml "hl_lines=1 7" >}}
 name: hello # service name
 environments:
-  - name: staging
-    path: "opta.yaml"
+- name: staging
+  path: "opta.yaml"
+  modules:
+- type: k8s-service
+  name: app # module name
+  port:
+  http: 80
+  image: ghcr.io/run-x/hello-opta/hello-opta:main
+  {{< / highlight >}}
+
+### Using Other Service Meshes
+Opta does not force Linkerd on its users, but merely makes it the default due to its 
+[clear merits, and popularity over other service meshes like Istio](https://linkerd.io/2022/02/16/linkerd-istio-adoption/index.html).
+An Opta user is free to use alternative service meshes by disabling the Linkerd installation in the `k8s-base` Opta module
+of their environment. This is done by adding the `linkerd_enabled: false` as a field like so:
+
+```yaml
+name: staging
+org_name: my-org
+providers:
+  aws:
+    region: us-east-1
+    account_id: XXXX # Your 12 digit AWS account id
 modules:
-  - type: k8s-service
-    name: app # module name
-    port:
-      http: 80
-    image: ghcr.io/run-x/hello-opta/hello-opta:main
-{{< / highlight >}}
+  - type: base
+  - type: k8s-cluster
+  - type: k8s-base
+    linkerd_enabled: false
+```
+
+A user is then responsible to make sure that the services are being added to the service mesh accordingly.
```

Comment:
they might not use service mesh at all. 
_If installing a different service mesh, a user is then responsible to make sure that the services are being configured accordingly._